### PR TITLE
[CLN] payment(_*): clean view hooks and split files per model

### PR DESCRIPTION
**Pull request by AntoineVDV**
When the payment views were updated with commit odoo/odoo@f7b8f075, a hook was improperly renamed to `code`, which doesn't help to figure out its purpose. This commit renames it to `provider_credentials` which better fits its role.

While doing so, the view files are also renamed and/or split by model to increase their readability.

See also:
- https://github.com/odoo/enterprise/pull/32490

### CODE REVIEW

File: addons/payment_authorize/views/payment_token_views.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="payment_token_form" model="ir.ui.view">
+        <field name='name'>Authorize.Net Token Form</field>
+        <field name='model'>payment.token</field>
+        <field name="inherit_id" ref="payment.payment_token_form"/>
+        <field name="arch" type="xml">
+            <xpath expr='//field[@name="provider_ref"]' position='after'>
```
Comment (Feyensv) [inline addons/payment_authorize/views/payment_token_views.xml:9]:
```suggestion
            <field name="provider_ref" position="after">
```

File: addons/payment_mollie/views/payment_provider_views.xml [hunk 2]
```diff
@@ -1,11 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+
     <record id="payment_provider_form" model="ir.ui.view">
         <field name="name">Mollie Provider Form</field>
         <field name="model">payment.provider</field>
         <field name="inherit_id" ref="payment.payment_provider_form"/>
         <field name="arch" type="xml">
-            <group name="code" position='inside'>
+            <group name="provider_credentials" position='inside'>
```
Comment (Feyensv) [inline addons/payment_mollie/views/payment_provider_views.xml:9]:
```suggestion
            <group name="provider_credentials" position="inside">
```
maybe harmonize single/double quotes in views too?

File: addons/payment_paypal/views/payment_transaction_views.xml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="payment_transaction_form" model="ir.ui.view">
+        <field name="name">PayPal Transaction Form</field>
+        <field name="model">payment.transaction</field>
+        <field name="inherit_id" ref="payment.payment_transaction_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='provider_reference']" position="after">
```
Comment (Feyensv) [inline addons/payment_paypal/views/payment_transaction_views.xml:9]:
```suggestion
            <field name="provider_reference" position="after">
```

File: addons/payment_authorize/views/payment_token_views.xml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="payment_token_form" model="ir.ui.view">
+        <field name='name'>Authorize.Net Token Form</field>
+        <field name='model'>payment.token</field>
```
Comment (Feyensv) [inline addons/payment_authorize/views/payment_token_views.xml:6]:
```suggestion
        <field name="name">Authorize.Net Token Form</field>
        <field name="model">payment.token</field>
```

**Comment by AntoineVDV** [comment]
@robodoo r+

**Comment by AntoineVDV** [comment]
@robodoo r+

**Pull request closed without merging**